src/thunderbots/software/CMakeLists.txt

```diff
@@ -250,7 +250,16 @@ if (CATKIN_ENABLE_TESTING)
 
     catkin_add_gtest(primitive_test
             test/primitive/primitive.cpp
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
This basically looks good to me, except for the merge conflicts in this file. Once we resolve those, should be good to go! Let me know if you want any help.